api/filter_outputs.go

```diff
@@ -0,0 +1,59 @@
+package api
+
+import (
+	"net/http"
+
+	"github.com/ONSdigital/dp-cantabular-filter-flex-api/model"
+	dprequest "github.com/ONSdigital/dp-net/v2/request"
+	"github.com/pkg/errors"
+)
+
+func (api *API) createFilterOutput(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+
+	if !dprequest.IsCallerPresent(ctx) {
```

Comment:
Sorry to mess around, I think this is actually not needed. Instead it we can just remove the public endpoint entirely. You will need to update your unauthorized test slightly (or maybe even remove it)